README.md

```diff
@@ -0,0 +1,18 @@
+# bioutils
+
+A repository for generally useful bioinformatics tools
+
+# Description
+
+Here, we will consolidate scripts for commonly-used functions such as converting filetypes or calculating descriptive statitics.
+
+# Usage
+
+1. Go to https://github.com/maragkakislab/bioutils and fork the repository to your own Github account
+2. Clone the repository into `~/dev/` with `git clone https://github.com/[username]/bioutils`
+3. Install all binaries into `~/bin/` with `for file in ~/dev/bioutils/*/bin/*;do ln -s $file ~/bin/$(basename "${file%.*}");done`
```

Comment:
I would suggest step number 3 would be to cd into the `bioutils` repository (wherever that is). Then step 4 can drop the `~/dev/bioutils` part from the command.